src/Core/Cache/ICacheDriver.php

```diff
@@ -21,28 +21,27 @@
 	public function get($key);
 
 	/**
-	 * Stores data in the cache identified by the key. The input $value can have multiple formats.
+	 * @brief Stores data in the cache identified by the key. The input $value can have multiple formats.
 	 *
 	 * @param string  $key      The cache key
 	 * @param mixed   $value    The value to store
-	 * @param integer $duration The cache lifespan, must be one of the Cache constants
+	 * @param integer $ttl 		 The cache lifespan, must be one of the Cache constants
```

Comment:
Please use spaces instead of tabs to align field list columns, they size differ between editors/display.